### ISSUE TITLE
Excalidraw window overlays pinned windows

### ISSUE THREAD

**Issue by reflector88** [philiphendry/ExcalidrawInVisualStudio#22]
**Description**
Pinned windows are covered by the Excalidraw window when they are opened. Unpinned windows work fine.

**Expected Behavior**
Pinned window should overlay the Excalidraw window.

**Actual Behavior**
The Excalidraw window overlays the pinned window.

**Screenshots**
![Screenshot 2024-06-17 145836](https://github.com/philiphendry/ExcalidrawInVisualStudio/assets/40728635/2f244a30-7fed-4610-90b5-79ae4f4bf9aa)

**Environment**
- Windows 11
- Visual Studio Community 2022 17.10.2

**Comment by philiphendry** [comment]
Thanks @reflector88 , appreciate the heads up. The problem appears to be with the WebView2 control and how it's rendering. I test swapped out the WebView2 for just a WPF button and, as expected, the rendering is fine:

![image](https://github.com/philiphendry/ExcalidrawInVisualStudio/assets/705830/fe31366a-59f4-45e7-8600-a9030beb2e0b)

I think it's something to do with the fact that WebView2 is rendering using the hwnd but that needs some more research.

As an alternative I tried swapping out the WebView2 for [CefSharp](https://cefsharp.github.io/) and this seemed to work but the size of the extensions vsix file had increased to 200mb (after I manually deleted some unnecessary bits) because it requires bundling CefSharp - so I'm going to scratch that idea!!

**Comment by philiphendry** [comment]
There's an open ticket that's been around for several years now but which finally sees some [development work](https://github.com/MicrosoftEdge/WebView2Feedback/issues/286#issuecomment-1937013149)! I've installed the nuget found at the link in the previous paragraph and swapped the WebView2 for a WebView2CompositionControl and it worked! I then tried the latest stable build (1.0.2592.51) but that doesn't include WebVIew2CompositionControl, but the prerelease (1.0.2646-prerelease) [does](https://learn.microsoft.com/en-us/dotnet/api/microsoft.web.webview2.wpf.webview2compositioncontrol?view=webview2-dotnet-1.0.2646-prerelease) and that solves this issue. Rather than release with a preview, I'm going to wait for the final release and then publish an update.
